src/app/modules/forms/ix-forms/components/ix-combobox/ix-combobox.component.ts

```diff
@@ -98,11 +98,16 @@ export class IxComboboxComponent implements ControlValueAccessor, OnInit {
 
   writeValue(value: string | number): void {
     this.value = value;
+
     if (!this.value) {
       this.selectedOption = null;
     }
     if (this.value && this.options?.length) {
-      this.selectedOption = { ...(this.options.find((option: Option) => option.value === this.value)) };
+      let existingOption = this.options.find((option: Option) => option.value === this.value);
+      if (!existingOption && this.allowCustomValue()) {
+        existingOption = { label: this.value as string, value: this.value };
+      }
+      this.selectedOption = { ...existingOption };
```

Comment:
Would be good to add a test case for changes in core components.

Reply:
sounds reasonable.